### PR TITLE
doc: udpate changelog

### PR DESCRIPTION
**Pull request by hhyasdf**
Pull Request Description
---

### Describe what this PR does / why we need it
Update change log.

### Special notes for reviews
For release v0.1.1.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,4 +16,16 @@ All notable changes to this project will be documented in this file.
...
 - Remove overlay logs for underlay-only mode
 - Fix error of using prefer interfaces list
-- Fix timeout error of pod creation on large scale
\ No newline at end of file
+- Fix timeout error of pod creation on large scale
+
+## v0.1.1
+### Improvements
+- Add checks for pod using the same subnet with node
+- Support setting lsinux kernel neigh gc thresh parameters
```
Comment (mars1024) [inline CHANGELOG.md:24]: lsinux --> linux

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,4 +16,16 @@ All notable changes to this project will be documented in this file.
...
+- Only choose vtep and node ip as node internal overlay container networking ip, support extra select
```
Comment (mars1024) [inline CHANGELOG.md:25]: select --> selection

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,4 +16,16 @@ All notable changes to this project will be documented in this file.
...
+- Remove duplicated routes
+- Adapt to underlay physical environment with arp sender ip check
+- Add checks for check pod network configuration, if not ready, pod will not be created successfully
```
Comment (mars1024) [inline CHANGELOG.md:28]: check --> prechecking

Comment (hhyasdf) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (hhyasdf) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (hhyasdf) [reply CHANGELOG.md]: fixed

Comment (hhyasdf) [reply CHANGELOG.md]: fixed

**Review by mars1024 — APPROVED** [review]
/lgtm

**Pull request merged**